charts/s3gw/questions.yaml

```diff
@@ -169,3 +204,11 @@ questions:
     required: false
     type: string
     group: "Advanced"
+
+  - variable: logLevel
+    default: "1"
+    description: "s3gw pod log level"
```

Comment:
This should either link to the docs with more info or at least give a hint of the possible values, e.g.:
"s3gw pod log level (1: lowest, 10: highest)"

Reply:
there is no upperbound; what do you think to put:
_s3gw pod log level, lower values are less verbose_